linode/manager
# test: [M3-8298] - Add cypress test coverage to DX tools additions in Linode create flow

## Description 📝
This is followup of #10547 Adds cypress test coverage to DX tools additions in Linode create flow. 


## Target release date 🗓️
7/8


## How to test 🧪

### Verification steps
(How to verify changes)
```
yarn cy:run -s "cypress/e2e/core/linodes/create-linode-view-code-snippet.spec.ts”
```

## As an Author I have considered 🤔

*Check all that apply*
